test/spec/features/modeling/behavior/AdaptiveLabelPositioningBehaviorSpec.js

```diff
@@ -270,51 +269,6 @@ describe('modeling/behavior - AdaptiveLabelPositioningBehavior', function() {
 
     describe('on label creation', function() {
 
-      describe('through <create.shape>', function() {
```

Comment:
Where did this test case go?

Reply:
This test case is not possible anymore. Setting `createElementsBehavior` will prevent the label position adjustment, not setting it will create an [additional label](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/modeling/behavior/LabelBehavior.js#L106).

Reply:
Got it. So the behavior does not ever apply on `create` anymore.

Create means we're pre-setting the positions and the feature is not supposed to fiddle around with it.